corgidrp/l2a_to_l2b.py

```diff
@@ -157,27 +157,39 @@ def flat_division(input_dataset, master_flat):
 
     return None
 
-def correct_bad_pixels(input_dataset):
+def desmear(input_dataset):
     """
-    
-    Compute bad pixel map and correct for bad pixels. 
-
-    MMB Notes: 
-        - We'll likely want to be able to accept an external bad pixel map, either 
-        from the CalDB or input by a user. 
-        - We may want to accept just a list of bad pixels from a user too, thus 
-        saving them the trouble of actually making their own map. 
-        - We may want flags to decide which pixels in the DQ we correct. We may 
-        not want to correct everything in the DQ extension
-        - Different bad pixels in the DQ may be corrected differently.
 
+    Desmear data frames.
```

Comment:
Please add more of a description as to what the function does

Reply:
Added. There's a question regarding the effective exposure time that results after applying desmearing. Once I know what to do about it, I can resubmit this PR, or work on any further step needed to deal with that

Reply:
It is an old comment that was inaccurate and does not apply any longer